docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2.1'
```

Comment:
First time I see this low version of compose schema in use, current is 3.8

Reply:
good catch. I'll fix that.